### PR TITLE
#56: Improved BaseColumnMetadataReader JDBC types mapping.

### LINKED ISSUE #56: Improve BaseColumnMetadataReader JDBC types mapping

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #56

### CODE REVIEW

File: src/main/java/com/exasol/adapter/jdbc/BaseColumnMetadataReader.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.exasol.adapter.jdbc;
 
 import static com.exasol.adapter.jdbc.RemoteMetadataReaderConstants.*;
+import static com.exasol.adapter.metadata.DataType.ExaCharset.UTF8;
```
Comment (jakobbraun) [inline src/main/java/com/exasol/adapter/jdbc/BaseColumnMetadataReader.java:4]: how about also including `DataType.ExaCharset.ASCII` ?

File: src/test/java/com/exasol/adapter/jdbc/BaseColumnMetadataReaderTest.java [hunk 2]
```diff
@@ -42,6 +42,21 @@ void testMappingUnsupportedTypesReturnsUnsupportedType(final int jdbcType) {
         assertThat(this.reader.mapJdbcType(jdbcTypeDescription).getExaDataType(), equalTo(ExaDataType.UNSUPPORTED));
     }
 
+    @ValueSource(ints = { Types.NUMERIC })
```
Comment (jakobbraun) [inline src/test/java/com/exasol/adapter/jdbc/BaseColumnMetadataReaderTest.java:45]: Why a parametrized test?

File: src/main/java/com/exasol/adapter/jdbc/BaseColumnMetadataReader.java (hunk 1 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/jdbc/BaseColumnMetadataReader.java:4]: done

File: src/test/java/com/exasol/adapter/jdbc/BaseColumnMetadataReaderTest.java (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/test/java/com/exasol/adapter/jdbc/BaseColumnMetadataReaderTest.java:45]: You're right. Forgot to refactor it after removing types

**Pull request merged**